apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -553,6 +558,11 @@ const Ticket = new GQLListSchema('Ticket', {
                 },
             },
         },
+        customType: {
```

Comment:
why not just `type` ?

Reply:
Type is too ambigious for my liking.. I really wanted to indicate that this will be used for some custom grouping or such

Maybe `group` would be nicer name, wyt?

Reply:
Decided to name it customClassifier